### ISSUE TITLE
write and submit grant application by july 31st

### ISSUE THREAD

**Issue by Hodlatoor** [ubiquity/partnerships#38]

**Comment by 0x4007** [comment]
It isn't clear to me why there are two submissions for this project.

**Comment by Hodlatoor** [comment]
They had two vectors for application

**Comment by 0x4007** [comment]
I am assuming that this can be done within an 8 hour window, but that it will likely be done over several days in short bursts. That's why I set the time to "<1 Day"

I suppose it might be more expressive to rename the `Time` labels to be in hours so that it does not imply over how many days, which is more aligned with how we've been using this system e.g. `Hours: <8`

Unfortunately people tend to conflate a time estimate in hours with an hourly rate, so this could cause issues with how its interpreted, even if the pricing algorithm nets the same amount in the end.

**Comment by Hodlatoor** [comment]
Takes a little bit longer than 8 hours bubba. Also, not everyone works the same way. Some sit for an uninterrupted set block. Some work styles involve stepping away from a task to jump back into it. That's also assuming we have all the pieces and do not need need for input from others or to create new task specific items.

**Comment by 0x4007** [comment]
The intent for the time estimate parameter is to estimate work input time, not waiting time.

**Comment by Hodlatoor** [comment]
I am aware of that. Waiting is just one component of what goes into a deliverable and it does go into the overall time use however in the context of what I mentioned above, it is not a primary time function. The other elements are.  There are different styles to solve the same problem and they all should be accounted for. #userempathy

**Comment by 0x4007** [comment]
> Waiting is just one component of what goes into a deliverable and it does go into the overall time use

No lol. Feel free to work on another task while you wait though. Sitting around waiting isn't productive.

**Comment by Hodlatoor** [comment]
@pavlovcik @sergfeldman 
To clarify some opened tasks by @sergfeldman, the next active phase I see for SAFE and DevPool is on SAFE DAO Forum:
https://forum.safe.global/c/proposals/21

 which is where we submit the grant proposal for SAFE GRANTS
 https://safe-global.notion.site/Safe-Grants-7a14a7403f1f44f0bfb236a7521dac91
 
 To reiterate, my idea here was a small grant which provides value for SAFE and easy for us to deploy for a small grant amount which kicks off the formal relationship: Integrate a safe into each bounty hunter account and each project manager. Reason being, a small grant is easy to approve especially considering their limited budget from grants. They get an immense value proposition for the grant amount since as the userbase grows, each user gets onboarded to SAFE and in-turn SAFE benefits immensely from this. Lastly -all the components we need to easily deploy this exist on our end.

@sergfeldman 
1. If you can keep an eye here with me on when the specs come out that will be great. Then I will finish writing up the grant and we can submit it.  
2. Do you want my login for SAFE DAO or want to add your own?

@pavlovcik when you can, please take a look at the repeating issues which were recently opened and close them if deemed duplicates so that we minimize confusion.

**Comment by 0x4007** [comment]
> @pavlovcik when you can, please take a look at the repeating issues which were recently opened and close them if deemed duplicates so that we minimize confusion.

Best if you tag @sergfeldman in each questionable issue and discuss one-by-one

**Comment by sergfeldman** [comment]
@Hodlatoor 

> If you can keep an eye here with me on when the specs come out that will be great. Then I will finish writing up the grant and we can submit it.

Sure, will do!

> Do you want my login for SAFE DAO or want to add your own?

I guess 1 SAFE DAO account is enough for submitting a grant.
If we find suitable that I have a separate account, I will create it.

> @pavlovcik when you can, please take a look at the repeating issues which were recently opened and close them if deemed duplicates so that we minimize confusion.

I believe that the task I created is not a duplicate of any grant task :)
This task was aimed at OTHER (not grant) forms of cooperation with SAFE.
- https://github.com/ubiquity/business-development/issues/50

**Comment by sergfeldman** [comment]
@Hodlatoor 
> Integrate a safe into each bounty hunter account and each project manager

If you are interested in my opinion, I think that the idea is wonderful for grant.

**Comment by Hodlatoor** [comment]
> @Hodlatoor 
> > Integrate a safe into each bounty hunter account and each project manager
> 
> If you are interested in my opinion, I think that the idea is wonderful for grant.

of course I am and thanks! that's what I want us to ship to them once they open up .

**Comment by Hodlatoor** [comment]
Update: they are opening this on july 27th.

**Comment by Hodlatoor** [comment]
keeping open

**Comment by Hodlatoor** [comment]
waiting on them, still.

**Comment by 0x4007** [comment]
Perhaps it makes sense to associate your grant application with this issue and close it. This "task operating system" focuses on work input and compensates for that. We can't control if they respond, I guess. 

But the task to "write and submit the grant application by July 31st" technically has been completed, presumably.

**Comment by Hodlatoor** [comment]
Sure. I can close it and keep an eye on them.

**Comment by 0x4007** [comment]
> associate your grant application with this issue and close it.

This means to open a pull request with your deliverable.

**Comment by Hodlatoor** [comment]
Right now I just have the intent which i have done previously they had a delay picking out their selection panel so now they pushed back when they open the submission program.

**Comment by 0x4007** [comment]
You need to finish the work, deliver it, and pass review to be eligible for bounties. If you don't complete this process then you are not eligible to collect bounties.

**Comment by Hodlatoor** [comment]
they pushed it back: Wave 1 Safe grants applications will open on the 16.08.2023.

**Comment by 0x4007** [comment]
> they pushed it back: Wave 1 Safe grants applications will open on the 16.08.2023.

Can you link your source?

**Comment by Hodlatoor** [comment]
updated previously

**Comment by 0x4007** [comment]
> > they pushed it back: Wave 1 Safe grants applications will open on the 16.08.2023.
> 
> 
> 
> Can you link your source?

Still waiting on this

**Comment by Hodlatoor** [comment]
submitted.
https://github.com/Hodlatoor/partnerships/blob/development/DevPool%20-%20The%20DAO%20OS%20and%20Talent%20Marketplace.md

**Comment by 0x4007** [comment]
> You need to finish the work, deliver it, and pass review to be eligible for bounties. If you don't complete this process then you are not eligible to collect bounties.

This did not pass review and should not have been marked as completed.

**Comment by Hodlatoor** [comment]
> > You need to finish the work, deliver it, and pass review to be eligible for bounties. If you don't complete this process then you are not eligible to collect bounties.
> 
> This did not pass review and should not have been marked as completed.



It was complete the moment it was submitted. And it was submitted. the md file I referenced above is just for our records. it is a copy of what was already submitted on their system. 

clarify.
